test/cpp_extensions/libtorch_agnostic_extension/setup.py

```diff
@@ -35,6 +36,16 @@ def get_extension():
     extra_compile_args = {
         "cxx": ["-fdiagnostics-color=always"],
     }
+    extra_link_args = []
+
+    # Note that adding this flag does not mean extension's parallel_for will
+    # always use OPENMP path, OpenMP path will only be used if (1) AND (2)
+    # (1) libtorch was built with OpenMP
+    # (2) extension compiles and links with -fopenmp
+    # macOS clang does not support -fopenmp so we need to skip it
```

Comment:
This is an incorrect statement, it does, option simply needs to be wrapped with `-Xcompiler -fopenmp`

And indeed, if we want some sort of an abstraction there, shouldn't we have helper function, say `torch.utils.cpp_extensions.get_openmp_flags()` (which I think already exist)